main/test/flix/Test.Assoc.Type.flix

```diff
@@ -16,4 +16,20 @@ mod Test.Assoc.Type {
         pub def head(x: String): Option[Char] = if (String.isEmpty(x)) None else Some(String.charAt(0, x))
     }
 
+    class Collection2[t] {
+        type Elm
+
+        pub def isEmpty(x: t): Bool
+        pub def contains(x: t, e: Collection2.Elm[t]): Bool
```

Comment:
Later I assume we will put `Elm` into the relevant scope?

Reply:
Yes. This is related to another issue where we have to qualify other signatures.

Reply:
Is this fixable or requires a lot of work?

Reply:
Fixable but will be a breaking change